src/base/static/components/atoms/typography.js

```diff
@@ -108,17 +109,18 @@ Header6.propTypes = {
   classes: PropTypes.string,
 };
 
-const PanelLabel = ({ children, ...props }) => {
-  return (
-    <span className={classNames("mapseed__panel-label", props.classes)}>
-      {children}
-    </span>
-  );
-};
+// TODO: Other label types.
+const RegularLabel = styled("label")(
```

Comment:
I think having a set of labels, like `SmallLabel`, `RegularLabel`, etc. is more semantic than using `SmallText`, `RegularText`, etc. for this purpose. I'd say the `*Text` components should be used for body text, and we should have `*Label` atoms for text whose purpose is to label an icon, swatch, or other graphical element.